e2e/babel-plugin-jest-hoist/package.json

```diff
@@ -7,6 +7,10 @@
   },
   "jest": {
     "automock": true,
-    "testEnvironment": "node"
+    "testEnvironment": "node",
+    "transformIgnorePatterns": [
+      "jest-jasmine2",
+      "jest-circus"
```

Comment:
Adding `jest-circus` as well so that when it becomes the default runner the tests won't fail